docs/source/distributed_simulation.rst

```diff
@@ -24,20 +24,21 @@ Architecture Overview
 Client-Server Model
 ===========================
 
-UE multiplayer is based around the client-server model. Each player is a client and connected to single server.
-In the robotics simulator, we put robots in the each client. 
+UE multiplayer is based around the client-server model. Each player is a client connected to a single server.
+In the robotics simulator, we put robots in each client. 
 
-.. image:: images/overview_of_UE_distributed_simulation.png
+.. figure:: images/overview_of_UE_distributed_simulation.png
 
-Fig shows an overview of the configuration. 
-The simulator consists of four types of instances. 
+   Figure 1: Overview of the configuration.
```

Comment:
is it possible to centering the caption?

Reply:
Yes sure, with the option `:align: center`. Unfortunately, this is not feasible with the TurtleBot video...